src/command.c

```diff
@@ -85,7 +85,7 @@ const command_s cmd_list[] = {
 	{"version", cmd_version, "Display firmware version info"},
 	{"help", cmd_help, "Display help for monitor commands"},
 	{"jtag_scan", cmd_jtag_scan, "Scan JTAG chain for devices"},
-	{"swdp_scan", cmd_swdp_scan, "Scan SW-DP for devices: [TARGET_ID]"},
+	{"swd_scan", cmd_swd_scan, "Scan SWD interface for devices: [TARGET_ID]"},
```

Comment:
For compatibility, we need to either keep the command named the same thing or implement a shadow version of the command named "swdp_scan" as otherwise we'll break scripts and automated workflows.

We personally agree with this change but need to think about the release cycle and how to do a deprecation here.

Reply:
Ah, right! I hadn't thought about that...

Reply:
I added a simple alias with "Deprecated" noted on the description, ideally we might want a "deeper" alias that also prints a warning

We can leave this out of this PR, let me know

Reply:
Think that works nicely till we figure out a better way to hang onto and implement deprecated commands so they don't get advertised in the help output but do still work (and print a warning on use).